### DISCUSSION TITLE
Erweiterung von User-Statuses-Endpoint um Datum

### DISCUSSION THREAD

**Discussion by bendix-dev** [Traewelling/traewelling#2180] [Ideas / Feature Requests]
Ich würde gerne beim Endpoint `/user/{username}/statuses` die Möglichkeit haben, ein Datum anzugeben, um in der Statusliste zu dem Datum springen zu können.

Was die Pagination angeht, hätte ich zwei Ideen:
1. Die page startet regulär bei 1, alle Status vor dem Datum kriegt man nicht raus.
2. Es wird zu der page gesprungen, die den ersten Status mit dem angegebenen Datum hat. Danach kann man vor und zurück, unabhängig vom Datum.

**Comment by marhei** [comment]
Statt API-seitig noch eine Paginierung hinzuzufügen wäre es doch viel sinnvoller der API sagen zu können ich will die ersten 15 Check-ins vor bzw. nach einem Datum und man setzt dann die Paginierung Clientseitig um indem man für die nächste Anfrage einfach das Datum des letzten (oder halt ersten, je nach Richtung) zurückgegeben Check-ins mitgibt.

So kann man auch verhindern, dass Check-ins doppelt oder gar nicht zurückgegeben werden, weil zwischen durch neue hinzugekommen sind oder welche gelöscht wurden.

**Reply by MrKrisKrisu** [reply]
Das ist im Prinzip hier bereits aufgeschrieben: #2041
Ich würde auch nach und nach die Cursor Pagination einbauen - das ist allerdings ein Breaking Change.
